modular_skyrat/modules/resleeving/code/rsd_interface.dm

```diff
@@ -0,0 +1,41 @@
+/obj/item/rsd_interface
+	name = "RSD brain interface"
```

Comment:
name = "RSD Phylactery"
desc = "A small device inserted, typically, into inert brains. As Resonance cannot persist in what's referred to as a 'vacuum' RSDs, much like the brains and CPUs they emulate, employ cerebral white noise as a foundation for Resonance to persist in otherwise dead-quiet containers."